components/rule-mgt/org.wso2.carbon.identity.rule.evaluation/src/main/java/org/wso2/carbon/identity/rule/evaluation/model/RuleEvaluationResult.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.rule.evaluation.model;
+
+/**
+ * Rule evaluation result.
+ * This class is used to represent the result of a rule evaluation.
+ */
+public class RuleEvaluationResult {
+
+    private final String ruleId;
+    boolean ruleSatisfied;
```

Comment:
Don't we need to make this private, as we have a constructer and a getter ?

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/6309/commits/3759ec3359d280decdbfa3e007b35654d0d7dccd